src/locale/translations/en.json

```diff
@@ -101,6 +101,7 @@
     "Platform-instagram": "Share to Instagram",
     "More": "More apps",
     "Message": "Join me in helping to slow the spread of COVID-19. Download the COVID Shield app: https://covidshield.app",
+    "InstagramImageUrl": "https://user-images.githubusercontent.com/5274722/84658989-b9ba6b00-aee4-11ea-84d4-d840527467b4.png",
```

Comment:
Sharing on Instagram iOS requires url, so I use github as the image host for now.